backend/app/policies/team_member/demandeur_policy.rb

```diff
@@ -1,5 +1,9 @@
 class TeamMember::DemandeurPolicy < TeamMemberPolicy
   def permitted_attributes
-    [:email]
+    [
+      :email,
+      :family_name,
```

Comment:
We need to make inherite `TeamMember::DemandeurPolicy` from `TeamMemberPolicy` as we would need to modify phone number or jobs as well.

Reply:
But this is already the case no ?

Reply:
It is now with your last commit and hour ago